pandas/tests/apply/test_numba.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+import pytest
+
+from pandas import DataFrame
+import pandas._testing as tm
+
+
+def test_numba_vs_python_noop(float_frame, apply_axis):
+    func = lambda x: x
+    result = float_frame.apply(func, engine="numba", axis=apply_axis)
+    expected = float_frame.apply(func, engine="python", axis=apply_axis)
+    tm.assert_frame_equal(result, expected)
+
+
+def test_numba_vs_python_indexing(float_frame):
+    row_func = lambda x: x["A"]
+    result = float_frame.apply(row_func, engine="numba", axis=1)
+    expected = float_frame.apply(row_func, engine="python", axis=1)
+    tm.assert_series_equal(result, expected)
+
+    row_func = lambda x: x["ZqgszYBfuL"]  # This is a label in the index
```

Comment:
Nit: I think it would be okay using a different fixture (specific to this file) than `float_frame`. IMO it can be a little more work to go back and debug not knowing what `float_frame` exactly is

Reply:
Sure, will do.